src/app/shared/users/user.services.ts

```diff
@@ -1,15 +1,21 @@
+import { USERS } from './user.mocked';
 import { Injectable } from '@angular/core';
 import { User } from './user.interface';
 
 @Injectable({ providedIn: 'root' })
 export class UserService {
-    user: User = {
-        id: 1,
-        firstName: 'Jack',
-        lastName: 'Developer'
-    };
+    user: User;
 
     getUser(): User {
         return this.user;
     }
+
+    setUser(user: User): void {
```

Comment:
It is possible to use `get`-ter/`set`-ter approach:
```
set user(user: User) {
  this._user = user;
}

get user(): User {
  return this._user;
}

```

Reply:
We have  conflicts between user property end geters